main.js

```diff
@@ -75,15 +75,26 @@ async function runEslint () {
   )
 
   const eslint = require(`${GITHUB_WORKSPACE}/node_modules/eslint`)
-  const cli = new eslint.CLIEngine()
+
+  try {
+    const cli = new eslint.ESLint()
+  } catch (e) {
+    throw `The ESLint Node.js API is not available. You're using \`eslint@${eslint.Linter.version}\`. Upgrade to 7+. (${e})`
+  }
```

Comment:
@danielma How's this look?

![image](https://user-images.githubusercontent.com/423741/143318387-f33615b0-f210-41f9-8a09-cd3f7a1130fe.png)

We're sort of assuming that the error here is `TypeError: eslint.ESLint is not a constructor`.

Reply:
Yeah, that looks perfect. Very helpful!